app/uk/gov/hmrc/teamsandrepositories/services/PersistingService.scala

```diff
@@ -66,11 +67,21 @@ case class PersistingService @Inject()(
           acc + (r.name -> r.copy(teams = trs.teamName :: r.teams))
         }
       }
-      allRepos      <- dataSource.getAllRepositoriesByName()
-      orphanRepos    = (allRepos -- reposWithTeams.keys).values
-      reposToPersist = reposWithTeams.values.toSeq ++ orphanRepos
-      _              = logger.info(s"found ${reposToPersist.length} repos")
-      count         <- persister.updateRepos(reposToPersist)
+      allRepos <- dataSource.getAllRepositoriesByName()
+      orphanRepos = (allRepos -- reposWithTeams.keys).values
+      reposToSeq = reposWithTeams.values.toSeq ++ orphanRepos
+      frontendServices <- serviceConfigsConnector.getFrontendServices()
```

Comment:
if we're only using frontendServices to test if it contains a given service name, it might be worth turning this into a Set, would probably be faster than a Seq